errorHandlers/apiErrors.js

```diff
@@ -29,6 +29,9 @@ const isMissingScopeError = err =>
   err.error &&
   err.error.category === 'MISSING_SCOPES';
 
+const isGatingError = err =>
```

Comment:
You can reuse `isSpecifiedError` here. Something along the lines of:
```js
const isGatingError = err => isSpecifiedError({ statusCode: 403, category: 'GATED' })
```

Reply:
Addressed in https://github.com/HubSpot/cli-lib/pull/17/commits/34f992c7c88781ab54ecb40bbcd2e1dc5b5f5033